microsoftgraph/microsoft-graph-explorer-v4
# Fix: Expand/Collapse toggling

## Overview

Fixes #1572
Getting started no longer automatically expands

### Testing Instructions
1. Load GE
2. Collapse 'Getting Started' category
3. Expand 'Applications' category
4. Select a sample query
5. Observe fix.
